.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
change it to weekly...or 14 days would be good option

Reply:
Dependabot currently supports the following interval values: "daily", "weekly", and "monthly". There is no specific interval value for every 14 days. I have changed it to weekly